src/constants.ts

```diff
@@ -36,6 +36,16 @@ export const CONTAINERS = [
     },
 ];
 
+/**
+ * Names of all the networks created by Docker Compose in this repository.
+ * Make sure to keep them up to date to always be able to remove them all using this service.
+ */
+export const NETWORK_NAMES = [
```

Comment:
QQ: can’t we get this dynamically? It’s a pain to have to update this constant every time we make modifications to the docker-compose configurations. Also, one might not know this and may introduce bugs if he forgets to update this.

Reply:
Btw, why not use `docker compose down`?

Reply:
@victor-yanev it's called, but this is an extra cleanup step. Alternatively to avoid the need for const `NETWORK_NAMES` we could rely that all networks we use and only them have the `hedera-` prefix and remove them that way.

Reply:
>  we could rely that all networks we use and only them have the hedera- prefix and remove them that way.

Understood, it will now function as described.